zerver/tests/test_templates.py

```diff
@@ -92,6 +92,14 @@ def test_markdown_tabbed_sections_extension(self) -> None:
         expected_html_sans_whitespace = expected_html.replace(" ", "").replace("\n", "")
         self.assertEqual(content_sans_whitespace, expected_html_sans_whitespace)
 
+    def test_markdown_tabbed_sections_missing_tabs(self) -> None:
+        template = get_template("tests/test_markdown.html")
+        context = {
+            "markdown_test_file": "zerver/tests/markdown/test_tabbed_sections_missing_tabs.md",
+        }
+        with self.assertRaises(ValueError):
+            template.render(context)
```

Comment:
It's best to include the exception string in the assertion, because that protects against the future introduction of minor Python bugs in the code that might happen to also raise a ValueError.

Reply:
Ack